Octo-Bot/python-web-bot/util/Driver/role.py

```diff
@@ -0,0 +1,38 @@
+def isAdmin(driver):
+    try:
+        driver.find_element_by_id('BodyContent_buttonLoginAdmin')
+        return True
+    except:
+        return False
+
+def isPatient(driver):
+    try:
+        driver.find_element_by_id('BodyContent_buttonLoginPatient')
+        return True
+    except:
+        return False
+
+def isResearcher(driver):
+    try:
+        driver.find_element_by_id('BodyContent_buttonLoginResearcher')
+        return True
+    except:
+        return False
+
+def isTherapist(driver):
+    try:
+        driver.find_element_by_id('BodyContent_buttonLoginTherapist')
+        return True
+    except:
+        return False
+
+def getRole(driver):
```

Comment:
Can we put it as a parameter when we run the docker image? So we can have several dockers running as patients and other dockers as therapists.

Reply:
Please put in the next PR!